config/nginx/django.conf.ctmpl

```diff
@@ -88,7 +120,6 @@ server {
 
     rewrite ^(/publications)(.*)$ /resources$2 permanent;
 }
-include /etc/nginx/conf.d/django_https;
```

Comment:
Shall we still be able to run on https without this?

Reply:
I added a function in the script that checks for the environment.. if the env is production it then appends it to the file...wanted to avoid manually removing it from staging
![env](https://user-images.githubusercontent.com/39744587/86117622-0a7eb600-bad8-11ea-8978-fb34167940d4.png)

Reply:
Nice 👍